.travis.yml

```diff
@@ -18,4 +18,4 @@ install: true
 script:
   - make .gitvalidation
   - make lint
-  - make binaries
+  - make ocid ocic
```

Comment:
@mrunalp sure, I rebased https://github.com/kubernetes-incubator/ocid/pull/50 also which should be pretty fast to merge since it's only client code - I'll then finally rebase this one and push force